docs/how-to-write-a-userstyle.md

```diff
@@ -8,6 +8,19 @@
 
 &nbsp;
 
+### Table of Contents
+
+<!--toc:start-->
```

Comment:
Are we using some sort of TOC generator with these comments?

Reply:
Yes via [marksman](https://github.com/artempyanykh/marksman)

Reply:
We should document that somewhere probably.

Reply:
probably contributing.md